src/Juvix/Compiler/Backend/Markdown/Error.hs

```diff
@@ -0,0 +1,51 @@
+module Juvix.Compiler.Backend.Markdown.Error where
+
+import Juvix.Compiler.Concrete.Language
+import Juvix.Compiler.Concrete.Translation.FromParsed.Analysis.Scoping.Error.Pretty
+import Juvix.Prelude
+
+data MarkdownBackendError
+  = ErrNoMarkdownInfo NoMarkdownInfoError
```

Comment:
It needs to be clear that this is an internal error

Reply:
Done